core/src/main/scala/cats/data/NonEmptySeq.scala

```diff
@@ -119,6 +119,11 @@ final class NonEmptySeq[+A] private (val toSeq: Seq[A]) extends AnyVal with NonE
    */
   def concat[AA >: A](other: Seq[AA]): NonEmptySeq[AA] = new NonEmptySeq(toSeq ++ other)
 
+  /**
+   * Append another `Seq` to this, producing a new `NonEmptySeq`.
+   */
+  def appendSeq[AA >: A](other: Seq[AA]): NonEmptySeq[AA] = concat(other)
```

Comment:
Seems we don't have a `NonEmptySeqSuite`, but at the least maybe we can add a test to the `SeqSuite`?

Reply:
Or even better would you like to create a `NonEmptySeqSuite`? :)

Reply:
Created the `NonEmptySeqSuite`. I copied some tests from the `SeqSuite` and added the new one.

The `FutureTestSuite` is failing although I didn't change anything in there. Any clues?

Reply:
Thanks, really appreciate that!

Really strange about the `Future` suite. I'll dig into a bit later, I'm intrigued.

Reply:
I just needed to merge `master` into this branch. Now the test is green and the PR awaiting your approval.